src/App.tsx

```diff
@@ -6,7 +6,7 @@ import Container from '@mui/material/Container/Container';
 
 export const App = () => {
   return (
-    <Container maxWidth='sm'>
+    <Container sx={{ width: '460px' }}>
```

Comment:
I'd say `maxWidth='sm'` is better as it uses a breakpoint from MUI so it's easier to customize it if needed.